proposals/0009-lockout-violation-detection.md

```diff
@@ -0,0 +1,172 @@
+---
+simd: '0009'
+title: Lockout Violation Detection
+authors:
+  - carllin
+  - ashwinsekar
+  - wencoding
+category: Standard
+type: Core
+status: Draft
+created: 2022-12-12
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+An algorithm designed to catch validators that violate lockout rules when
+voting.
+
```

Comment:
Maybe it is very naive but why couldn't a user/validator just post a proof of a vote on some other fork?

Reply:
I think the answer to this will enlighten me enough to actually review this proposal

Reply:
yes for slashing that is enough. the general idea is the slashing program will take:
* vote from current fork or alternate fork indicating some lockout conditions
* vote from current fork that violates said conditions
* some ancestory details showing the fork relationship of the two votes